android/src/com/mapswithme/util/bottomsheet/MenuBottomSheetFragment.java

```diff
@@ -11,64 +12,42 @@
 import androidx.fragment.app.Fragment;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
-
 import com.google.android.material.bottomsheet.BottomSheetBehavior;
 import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
 import com.mapswithme.maps.R;
-import com.mapswithme.util.ThemeUtils;
+import com.mapswithme.util.UiUtils;
 
 import java.util.ArrayList;
 
 public class MenuBottomSheetFragment extends BottomSheetDialogFragment
 {
+
   @Nullable
-  private final String title;
+  private ArrayList<MenuBottomSheetItem> mMenuBottomSheetItems;
   @Nullable
-  private final Fragment headerFragment;
-  private final ArrayList<MenuBottomSheetItem> menuBottomSheetItems;
-
-  public MenuBottomSheetFragment(@NonNull Fragment headerFragment, ArrayList<MenuBottomSheetItem> menuBottomSheetItems)
-  {
-    this.title = null;
-    this.headerFragment = headerFragment;
-    this.menuBottomSheetItems = menuBottomSheetItems;
-  }
-
-  public MenuBottomSheetFragment(@NonNull String title, ArrayList<MenuBottomSheetItem> menuBottomSheetItems)
-  {
-    this.title = title;
-    this.headerFragment = null;
-    this.menuBottomSheetItems = menuBottomSheetItems;
-  }
+  private Fragment mHeaderFragment;
 
-  public MenuBottomSheetFragment(ArrayList<MenuBottomSheetItem> menuBottomSheetItems)
-  {
-    this.title = null;
-    this.headerFragment = null;
-    this.menuBottomSheetItems = menuBottomSheetItems;
-  }
-
-  public MenuBottomSheetFragment(@NonNull Fragment headerFragment)
+  public MenuBottomSheetFragment()
   {
-    this.title = null;
-    this.headerFragment = headerFragment;
-    this.menuBottomSheetItems = new ArrayList<>();
   }
 
-  // https://stackoverflow.com/questions/51831053/could-not-find-fragment-constructor
```

Comment:
Why did you remove the comment?

Reply:
It is not relevant as the fragment has no constructor anymore.